client/handle_sign_in_callback.go

```diff
@@ -10,7 +10,7 @@ import (
 
 func (logtoClient *LogtoClient) HandleSignInCallback(request *http.Request) error {
 	signInContext := SignInContext{}
-	parseSignInContextErr := json.Unmarshal([]byte(logtoClient.storage.GetItem("logto_sign_in_context")), &signInContext)
+	parseSignInContextErr := json.Unmarshal([]byte(logtoClient.storage.GetItem(StorageKeySignInContext)), &signInContext)
```

Comment:
In other SDKs, this is called "sign in session"?

Reply:
Good, add an issue to align this, for the "context" is usage in some test descriptions.
LOG-4046